src/object-properties.js

```diff
@@ -42,9 +42,9 @@ const properties = {
      */
     font: {
       /**
-       * @type {string}
+       * @type {number}
        */
-      size: '',
+      size: 50,
```

Comment:
Not sure about the default, but 50 (%) looks a bit better than 100 I think

Reply:
Looks good to me to, has UX any thoughts?

Reply:
Not sure, but I will show it to Christofer tomorrow and ask what he thinks.

Reply:
Christofer is also good on 50%!